src/aiperf/dataset/loader/baseten_trace.py

```
@@ -0,0 +1,314 @@
+    def can_load(
+        cls, data: dict[str, Any] | None = None, filename: str | Path | None = None
+    ) -> bool:
+        if filename is None or Path(filename).suffix.lower() != ".parquet":
+            return False
+
+        if data is not None:
+            return _REQUIRED_COLUMNS.issubset(data.keys())
+
+        try:
+            schema = pq.read_schema(filename)
+        except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)